desk/app/channels.hoon

```diff
@@ -10,14 +10,147 @@
 ::
 /-  c=channels, g=groups, ha=hark, activity
 /-  meta
-/+  default-agent, verb, dbug, sparse, neg=negotiate, imp=import-aid, logs
+/+  default-agent, verb, dbug,
+    neg=negotiate, discipline, logs,
+    sparse, imp=import-aid
 /+  utils=channel-utils, volume, s=subscriber, em=emojimart
 ::  performance, keep warm
 /+  channel-json
 ::
-::  compile all marks
-::XX  /~ rune is broken, see urbit issue #5242
-:: /~  marks  *  /mar/channel
```

Comment:
It would be really nice if we could avoid having to list down every single mark by hand and feed /lib/discipline.hoon such a map. But the bug in question probably remains unfixed...

Reply:
agreed haha